presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCachingFileSystem.java

```diff
@@ -79,6 +85,16 @@ public FSDataInputStream openFile(Path path, HiveFileContext hiveFileContext)
                     .setPath(path.toString())
                     .setFolder(false)
                     .setLength(hiveFileContext.getFileSize().get());
+            if (cacheQuotaEnabled) {
+                CacheScope scope = CacheScope.create(hiveFileContext.getCacheQuota().getIdentity());
+                info.setCacheScope(scope);
+                if (hiveFileContext.getCacheQuota().getQuota().isPresent()) {
+                    info.setCacheQuota(new CacheQuota(ImmutableMap.of(scope.level(), hiveFileContext.getCacheQuota().getQuota().get().toBytes())));
+                }
+                else {
+                    info.setCacheQuota(CacheQuota.UNLIMITED);
```

Comment:
IIUC unlimited quota is the default behavior. How about:
```
if (cacheQuotaEnabled && hiveFileContext.getCacheQuota().getQuota().isPresent()) {
  // do cache quota magic
}
```

Reply:
not default in Alluxio. so I set it explicitly.
note that this is the data structure passed to Alluxio land

Reply:
Question: what is the default behavior if not set?

Reply:
Quota check will never fail. The behavior will be exactly the same as today

Reply:
then shouldn't it be the same as UNLIMITED?